### PR TITLE
[PAPI-358] Track Interflux token owner

### PR DESCRIPTION
**Pull request by drmathias**

### CODE REVIEW

**Review by mrtpain — APPROVED** [review]
LGTM 

You could have created the new sql script to drop the `token_chain` and create if not exists the new `token_wrapped`. 😉

File: pipeline-scripts/create-database.sql [hunk 1]
```diff
@@ -820,20 +820,29 @@ CREATE PROCEDURE CreateDatabase ()
...
 
-        CREATE TABLE IF NOT EXISTS token_chain(
+        CREATE TABLE IF NOT EXISTS token_wrapped(
             Id                  BIGINT UNSIGNED AUTO_INCREMENT,
             TokenId             BIGINT UNSIGNED NOT NULL,
+            Owner               VARCHAR(50) NOT NULL,
             NativeChainTypeId   SMALLINT UNSIGNED NOT NULL,
             NativeAddress       VARCHAR(100) NULL,
+            CreatedBlock        BIGINT UNSIGNED NOT NULL,
+            ModifiedBlock       BIGINT UNSIGNED NOT NULL,
```
Comment (mrtpain) [inline pipeline-scripts/create-database.sql:830]: Should we add the `Trusted` bit flag?

Comment (drmathias) [reply pipeline-scripts/create-database.sql:830]: Potentially, or we create a `trusted_custodian` table? There might be more than 1 and would save us having to update the database if it changes

Comment (mrtpain) [reply pipeline-scripts/create-database.sql:830]: Now I feel like we're pulling a thread that will keep going 😅 I guess we should skip for now and reevaluate once we get this first step done. 

We check at runtime within the token assemblers to compare the configurations against the owner to return a `Trused` flag if they match.

Comment (drmathias) [reply pipeline-scripts/create-database.sql:830]: I'll do what you suggested for now and simply use the config to create a trusted flag. You're right about the thread continuing, as something that occurred to me is that if we want to future-proof it (avoiding breaking changes in the API), we should consider additional wrapping methods alongside Interflux. In which case we might want to also return out to the API `bridgeType: Interflux` and we'd need different 'trusted' addresses for each bridge type.

**Comment by drmathias** [comment]
Had to amend slightly as it worked locally but not against DEVNET, seems to be because a versioning difference. Ran against DEVNET now

**Review by mrtpain — APPROVED** [review]

**Pull request merged**